src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,38 @@
+package ladder.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Line {
+
+    private final List<Boolean> points;
+
+    public Line (List<Boolean> points) {
+        this.points = points;
+    }
+
+    public boolean hasRightMoved(int index) {
+        if(index == (points.size()-1)) {
```

Comment:
```suggestion
        if (index == (points.size() - 1)) {
```
[java convetion](https://myeonguni.tistory.com/1596)을 확인 해주세요! 😄